feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -93,7 +94,12 @@ export class UpdatePasswordComponentService {
     });
   }
 
-  protected onError(_error: Error): void {
+  protected onError(_error: HttpErrorModel): void {
+    if (_error.details?.[0].type === 'AccessDeniedError')
+    this.globalMessageService.add(
```

Comment:
code not aligned, and always use {} even we only have one line inside the if

Reply:
strange, it worked as expected. Indeed I missed the {}, I will add it. Nice observation